recipes-core/images/iot2050-image-example.bb

```diff
@@ -60,6 +60,7 @@ IOT2050_DEBIAN_DEBUG_PACKAGES = " \
     net-tools \
     i2c-tools \
     sudo \
+    parted \
```

Comment:
Belongs as dependency into the package that needs it. Fold it over there into the patch that adds the feature.

Reply:
OK. Have added this package belong to the feature patch.

Reply:
I'm not seeing this yet. It's still only in the image recipe, not in DEBIAN_DEPENDS of install-on-emmc.